.husky/validate-commit-msg.js

```diff
@@ -3,7 +3,7 @@ const commitMsgFile = process.argv[2]
 const commitMsg = fs.readFileSync(commitMsgFile, "utf8").trim()
```

Comment:
Do we still have this file in repo? I couldn't see it after we migrated to commitlint 😕